src/eda_covid19.r

```diff
@@ -0,0 +1,88 @@
+# author: Fatime Selimi, Neel Phaterpekar, Nicholas Wu, Tanmay Sharma
+# date: 2020-11-25
+
+"Creates eda table and plots for the response ratio from the COVID-19 data (from https://github.com/owid/covid-19-data/tree/master/public/data).
+Saves the plots as a pdf and png file.
+Usage: src/eda_covid19.r --input_path=<input_path> --out_dir=<out_dir>
+  
+Options:
+--input_path=<input_path>     Path (including filename) to processed data in csv format
+--out_dir=<out_dir>           Path to directory where the plots should be saved
+" -> doc
+
+library(tidyverse)
+library(knitr)
+library(infer)
+library(cowplot)
+library(docopt)
+library(ggthemes)
+
+opt <- docopt(doc)
+
+main <- function(input_path, out_dir) {
+  
+    #create a table summarizing the key statistics
+    covid_CAN_USA <- read_csv(input_path)
+    
+    covid_CAN_USA_summary <-  covid_CAN_USA %>%
```

Comment:
I actually thought that the summary table should be part of script #3 instead of script #2, because: 

- M2 instruction says script #2 "reads the data from the first script and performs and data cleaning/pre-processing, transforming, and/or paritionting that needs to happen before exploratory data analysis or modeling takes place".  It didn't really mention doing a summary statistics. 

- Script# 3 "creates exploratory data visualization(s) **_and table(s)_** that are useful to help the reader/consumer understand that dataset."

Thoughts?

Reply:
I guess it depends on how one interprets this. 

The way I understand it is that we need to do all the data transformation in script 2 and then use the output/s of script 2 in our further data exploration and analysis. 

Plus, we're still creating the tables in script 3 using the transformed output from script 2, in this case, `covid_can_usa_summary.csv`.

@tanmaysharma19 @nphaterp what do you guys think?